src/cmds.c

```diff
@@ -772,6 +775,61 @@ static void cmd_pls_bot(struct userrec *u, int idx, char *par)
     return;
   }
 
+#ifndef IPV6
+  if (!inet_pton(AF_INET, addr, saddr)) {
+    for (i = 0; addr[i]; i++) {
+      if (addr[i] == ':') {
+        dprintf(idx, "Invalid IP address format (this Eggdrop "
+          "was compiled without IPv6 support).\n");
+        return;
+      }
+    }
+  }
+#endif
+
+#ifndef TLS
+  if ((*port == '+') || (relay && (relay[1] == '+'))) {
+    dprintf(idx, "Ports prefixed with '+' are not enabled "
+      "(this Eggdrop was compiled without TLS support).\n");
+    return;
+  }
+#endif
+  if ((atoi(port) < 1) || (atoi(port) > 65535)) {
+    dprintf(idx, "Ports must be integers between 1 and 65535.\n");
+    return;
+  }
+  if (relay) {
+    relay2 = relay;
+    /* Convert to just the port number string for error checking */
+    relay2++;
+    if (*relay2 == '+') {
+      relay2++;
+    }
+    if ((atoi(relay2) < 1) || (atoi(relay2) > 65535)) {
+      dprintf(idx, "Ports must be integers between 1 and 65535.\n");
+      return;
+    }
+  }
+
+  if (*addr == '+') {
```

Comment:
Move this to the for-loop below to catch `.+bot Bot 1234/+4321`

Reply:
Done